src/.env-sample

```diff
@@ -36,4 +36,12 @@ SLACK_VERIFICATION_TOKEN=         # from `Basic Information` page of your Slack
 # AWS_SECRET_KEY=                   # AWS Secret Key
 
 # Github. Optional.
-# GITHUB_AUTH_TOKEN=                # Authorization token for Github API
\ No newline at end of file
+# GITHUB_AUTH_TOKEN=                # Authorization token for Github API
+
+#Hugchat
```

Comment:
Thanks for adding the comments, they're very helpful.

Let's have them all go on the same line as the flag option, like this:
`#HUGCHAT_EMAIL =                      #Your HuggingChat.com user account email address`